_pages/laws/508.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "508"
+---
+
+_Bristow and Yuda are good contacts for this and they are developing training._ See the [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/) for a checklist.
```

Comment:
Could we pull individual names out the guide, and point people to other guides and Slack channels?